src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -147,6 +150,21 @@ const sidebarWidgets = {
       };
     },
   },
+  fieldWidget: {
+    className: 'widgets-value',
+    template: hbs`{{ displayValue }}{{#unless displayValue}}{{{ defaultHtml }}}{{/unless}}`,
+    templateContext() {
+      const defaultHtml = this.getOption('default_html');
+      const displayValue = getFieldValue(this.model.getFields(), this.getOption('field_name'), this.getOption('key'));
+
+      if (!displayValue) return { defaultHtml };
```

Comment:
So we can remove this check.  It isn't doing anything in this widget.  We can probably go ahead and remove it everywhere and it'd be just fine.  Can you test that?